### PR TITLE
feat: polling default ns

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

File: core/src/main/java/io/javaoperatorsdk/jenvtest/KubeAPIServer.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class KubeAPIServer implements UnexpectedProcessStopHandler {
 
   private static final Logger log = LoggerFactory.getLogger(KubeAPIServer.class);
 
-  public static final int STARTUP_TIMEOUT = 15_000;
+  public static final int STARTUP_TIMEOUT = 10_0000;
```
Comment (metacosm) [inline core/src/main/java/io/javaoperatorsdk/jenvtest/KubeAPIServer.java:15]: 100 seconds? That seems too much to wait.

Comment (csviri) [reply core/src/main/java/io/javaoperatorsdk/jenvtest/KubeAPIServer.java:15]: yeah that is a mistake

**Comment by csviri** [comment]
replaced by: https://github.com/java-operator-sdk/jenvtest/pull/56

**Comment by metacosm** [comment]
Should we remove this branch? And maybe activate automated branch removal on merges?

**Comment by csviri** [comment]
yeah, deleted, will activate it.

**Pull request closed without merging**